site/themes/arangodb-docs-theme/layouts/partials/shortcodes/expand.html

```diff
@@ -6,19 +6,21 @@
 {{- if eq (printf "%T" $expanded) "string" }}
 	{{- $expanded = (eq $expanded "true") }}
 {{- end }}
-{{- with $context }}
-<div class="expand
-    {{- if $expanded }} expand-expanded{{ end -}}
-">
-    {{/* things are getting complicated when search tries to open the expand box while jquery sets the display CSS on the element */}}{{ "" -}}
-    <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
-        {{ $title }}
-    </a>
-    <div class="expand-content" style="display:
-        {{- if $expanded }} block{{ else }} none{{ end -}}
-    ;">
-{{ $description }}
-{{ $content | safeHTML }}
+{{- with $content }}
+    {{- with $context }}
+    <div class="expand
+        {{- if $expanded }} expand-expanded{{ end -}}
+    ">
+        {{/* things are getting complicated when search tries to open the expand box while jquery sets the display CSS on the element */}}{{ "" -}}
+        <a class="expand-label" onclick="$t=$(this); if($t.parent('.expand-expanded.expand-marked').length){ $t.next().css('display','none') }else if($t.parent('.expand-marked').length){ $t.next().css('display','block') }else{ $t.next('.expand-content').slideToggle(100); } $t.parent().toggleClass('expand-expanded');">
```

Comment:
It would be better to avoid inline JS and register an event handler somewhere further up the DOM tree. Also, jQuery seems like an unnecessary dependency these days.